states/PA/governor.yaml

```diff
@@ -0,0 +1,203 @@
+            "bucks": "Bucks"
+            "butler": "Butler"
+            "cambria": "Cambria"
+            "cameron": "Cameron"
+            "carbon": "Carbon"
+            "centre": "Centre"
+            "chester": "Chester"
+            "clarion": "Clarion"
+            "clearfield": "Clearfield"
+            "clinton": "Clinton"
+            "columbia": "Columbia"
+            "crawford": "Crawford"
+            "cumberland": "Cumberland"
+            "dauphin": "Dauphin"
+            "delaware": "Delaware"
+            "elk": "Elk"
+            "erie": "Erie"
+            "fayette": "Fayette"
+            "forest": "Forest"
+            "franklin": "Franklin"
+            "fulton": "Fulton"
+            "greene": "Greene"
+            "huntingdon": "Huntingdon"
+            "indiana": "Indiana"
+            "jefferson": "Jefferson"
+            "juniata": "Juniata"
+            "lackawanna": "Lackawanna"
+            "lancaster": "Lancaster"
+            "lawrence": "Lawrence"
+            "lebanon": "Lebanon"
+            "lehigh": "Lehigh"
+            "luzerne": "Luzerne"
+            "lycoming": "Lycoming"
+            "mckean": "McKean"
+            "mercer": "Mercer"
+            "mifflin": "Mifflin"
+            "monroe": "Monroe"
+            "montgomery": "Montgomery"
+            "montour": "Montour"
+            "northampton": "Northampton"
+            "northumberland": "Northumberland"
+            "perry": "Perry"
+            "philadelphia": "Philadelphia"
+            "pike": "Pike"
+            "potter": "Potter"
+            "schuylkill": "Schuylkill"
+            "snyder": "Snyder"
+            "somerset": "Somerset"
+            "sullivan": "Sullivan"
+            "susquehanna": "Susquehanna"
+            "tioga": "Tioga"
+            "union": "Union"
+            "venango": "Venango"
+            "warren": "Warren"
+            "washington": "Washington"
+            "wayne": "Wayne"
+            "westmoreland": "Westmoreland"
+            "wyoming": "Wyoming"
+            "york": "York"
+    - fill_in:
+        - name: "zip"
+          selector: "#nf-field-1411"
+          value: $ADDRESS_ZIP5
+          required: true
+          # max length: 5
+        - name: "email"
+          selector: "#nf-field-1421"
+          value: "$EMAIL"
+          required: true
+        - name: "phone"
+          selector: "#nf-field-1431"
+          value: $PHONE
+          required: true
+        - name: "nf-field-1461"
+          selector: "#nf-field-1461"
+          value: "$MESSAGE"
+          required: true
+          # max length: 5000
+    - recaptcha:
+        - value: true
+    - click_on:
+        - value: "Submit"
+          selector: "#nf-field-1471"
+    # The submit button on the page changes to "processing" for a substantial length of time.  An unscientific test averaged about 10 seconds before the success message appears.  A 20 second delay seems to be a reasonable compromise between speed and ensuring success.
```

Comment:
This time can probably be shortened. By default, we wait 10 seconds for the text to appear in the body, so this additional wait can probably be 10 seconds or so.

Reply:
Is the "wait" time in addition to the default wait time?  If so, yeah, 10 + 10 would be what I was actually aiming for.

Reply:
Yep, it is. It's https://github.com/resistbot/mercury/blob/master/mercury/form_processor.py#L294.

Reply:
Oh ok.  I got you.  The wait is a separate instruction above the default.  I'll change it to 10.